examples/async_await/bin/futures_intro.dart

```diff
@@ -0,0 +1,27 @@
+// #docregion ''
+Future<void> getUserOrder() =>
+    // Imagine that this function fetches user info from a data source
```

Comment:
We should use the same comments as the gist, no?
(Except that comments should end in periods!)

Reply:
Reverted the text and added a period.